apps/client/src/features/rundown/event-block/EventBlock.utils.ts

```diff
@@ -39,3 +40,29 @@ export function formatOverlap(timeStart: number, previousStart?: number, previou
   const overlapString = formatDuration(Math.abs(timeFromPrevious), false);
   return `${timeFromPrevious < 0 ? 'Overlap' : 'Gap'} ${overlapString}`;
 }
+
+export function calculateAccumulatedGap(
```

Comment:
question: would it not be simpler for the accumulated gap to always be a number?